genesis_core/user_api/iam/api/controllers.py

```diff
@@ -47,6 +88,27 @@ def enforce(self, rule, do_raise=False, exc=None):
         return iam.enforcer.enforce(rule, do_raise, exc)
 
 
+class AltchaMixin:
```

Comment:
I propose to use straightforward name instead of implementation-specific name
```suggestion
class CaptchaMixin:
```